spec/features/course_page_spec.rb

```diff
@@ -43,6 +45,7 @@ def js_visit(path, count=3)
                     timeline_end: course_end.to_date,
                     school: 'This university.foo',
                     term: 'term 2015',
+                    home_wiki_id: es_wiktionary.id,
```

Comment:
This is what is changing the behavior of the later test, and making it no longer test what it was designed for. That test is supposed to be for how article titles show up when they are a different home wiki than the course. Previously, the home wiki relied on the default of en.wikipedia.

I suggest making it explicitly en.wikipedia here, so that no changes to other tests will be needed.